### PR TITLE
Image view fix

### PR DESCRIPTION
**Pull request by metehancelik**

### CODE REVIEW

**Comment by fatihalp** [comment]
@metehancelik 

Conflicting files

File: addons/default/visiosoft/advs-module/resources/views/list/gallery.twig [hunk 1]
```diff
@@ -24,12 +24,26 @@
...
+                            </p>
+                            -->
                         {% endif %}
                         {% if showPrice %}
-                            <div class="product-price-hb rounded d-flex align-items-center px-lg-3 px-2 mt-2 text-dark justify-content-between">
+                            <div class="product-price-hb rounded d-flex align-items-center mt-2 bg-secondary-md bg-none text-dark justify-content-between">
                                 {{ adv.price != '0' ? currency_format(adv.price,adv.currency) : trans('visiosoft.module.advs::field.free') }}
                                 {{ addBlock('list/extra-actions', {'ad': adv, 'vars': _context}, ['comparisons'])|raw }}
                             </div>
+                            <p class="pb-1">Bu ürünün farklı seçenekleri vardır.</p>
```
Comment (muammertop) [inline addons/default/visiosoft/advs-module/resources/views/list/gallery.twig:37]: no static value allowed

File: addons/default/visiosoft/advs-module/resources/views/list/gallery.twig [hunk 2]
```diff
@@ -24,12 +24,26 @@
...
+                            <div class="review">
+                                <!--Comment
```
Comment (muammertop) [inline addons/default/visiosoft/advs-module/resources/views/list/gallery.twig:39]: remove comment pls

**Review by muammertop — CHANGES_REQUESTED** [review]

File: addons/default/visiosoft/advs-module/resources/views/list/gallery.twig [hunk 3]
```diff
@@ -24,12 +24,20 @@
...
+                            -->
                         {% endif %}
                         {% if showPrice %}
-                            <div class="product-price-hb rounded d-flex align-items-center px-lg-3 px-2 mt-2 text-dark justify-content-between">
+                            <div class="product-price-hb rounded d-flex align-items-center mt-2 bg-secondary-md bg-none text-dark justify-content-between">
                                 {{ adv.price != '0' ? currency_format(adv.price,adv.currency) : trans('visiosoft.module.advs::field.free') }}
                                 {{ addBlock('list/extra-actions', {'ad': adv, 'vars': _context}, ['comparisons'])|raw }}
                             </div>
+                            <div class="review">
+                                <button class="m-0 btn btn-sm btn-outline-dark d-md-none d-block w-100">Sepete Ekle</button>
```
Comment (muammertop) [inline addons/default/visiosoft/advs-module/resources/views/list/gallery.twig:38]: ?

**Review by muammertop — CHANGES_REQUESTED** [review]

File: addons/default/visiosoft/advs-module/resources/views/list/gallery.twig (hunk 3 above)
Comment (metehancelik) [reply addons/default/visiosoft/advs-module/resources/views/list/gallery.twig:38]: using trans now

File: addons/default/visiosoft/advs-module/resources/views/list/gallery.twig (hunk 2 above)
Comment (metehancelik) [reply addons/default/visiosoft/advs-module/resources/views/list/gallery.twig]: removed

File: addons/default/visiosoft/advs-module/resources/views/list/gallery.twig (hunk 1 above)
Comment (metehancelik) [reply addons/default/visiosoft/advs-module/resources/views/list/gallery.twig]: removed

File: addons/default/visiosoft/advs-module/resources/views/list/gallery.twig [hunk 4]
```diff
@@ -24,12 +24,17 @@
...
                         {% endif %}
                         {% if showPrice %}
-                            <div class="product-price-hb rounded d-flex align-items-center px-lg-3 px-2 mt-2 text-dark justify-content-between">
+                            <div class="product-price-hb rounded d-flex align-items-center mt-2 bg-secondary-md bg-none text-dark justify-content-between">
                                 {{ adv.price != '0' ? currency_format(adv.price,adv.currency) : trans('visiosoft.module.advs::field.free') }}
                                 {{ addBlock('list/extra-actions', {'ad': adv, 'vars': _context}, ['comparisons'])|raw }}
                             </div>
+                            <div class="review">
+                                <button class="m-0 btn btn-sm btn-outline-dark d-md-none d-block w-100">{{ trans('visiosoft.module.advs::field.add_to_cart') }}</button>
+                            </div>
```
Comment (diashalabi) [inline addons/default/visiosoft/advs-module/resources/views/list/gallery.twig:36]: Use addBlock

**Review by diashalabi — CHANGES_REQUESTED** [review]

File: addons/default/visiosoft/advs-module/resources/views/list/gallery.twig (hunk 4 above)
Comment (metehancelik) [reply addons/default/visiosoft/advs-module/resources/views/list/gallery.twig:36]: @fatihalp how am I supposed to use "addBlock"?

Comment (fatihalp) [reply addons/default/visiosoft/advs-module/resources/views/list/gallery.twig:36]: explained

Comment (fatihalp) [reply addons/default/visiosoft/advs-module/resources/views/list/gallery.twig:36]: actually at that stage we dont need add to card here and not ready for it @metehancelik 

you can remove it

Comment (metehancelik) [reply addons/default/visiosoft/advs-module/resources/views/list/gallery.twig:36]: alright, will do it.

**Review by diashalabi — APPROVED** [review]

**Pull request merged**